### DISCUSSION TITLE
Ubuntu 24 support

### DISCUSSION THREAD

**Discussion by jpSimkins** [streamdeck-linux-gui/streamdeck-linux-gui#215] [Q&A]
I was going to post a bug report but before I do I wanted to see if anyone else experienced this issue and could help collect better details for the bug report, if needed.

I have a fresh install of Ubuntu 24. 

I have been using this guide for installation: 
- https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/blob/main/docs/installation/ubuntu.md

Then this to make it a service:
- https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/blob/main/docs/installation/systemd.md

The only variance is that i am doing:

```
python3 -m venv .venv
source .venv/bin/activate
python3 -m pip install streamdeck-linux-gui
```

After I install this, the OS takes a few mintues to respond to anything and the theme went from dark mode to light mode. After 10-20 mins, the theme seems to work again but every new boot, after I login, its the same experience. Opening finder takes a few mintues too. Only on the first boot. 

Uninstalling this gets the system back to working order.

First, I wonder if anyone else has experienced these issues?

I will try to collect more information for a formal bug report but looking to see if anyone else has already figured this out or perhaps there is a bug report already for the underlying issue here.

Thank you.

**Comment by coolapso** [comment]
Hi @jpSimkins even if this is a common issue, it will probably never get fixed. Please read https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203

thank you

**Reply by jpSimkins** [reply]
Thank you! I'm checking out the new project!
